src/more_ops.rs

```diff
@@ -134,7 +134,11 @@ pub fn op_unknown<A: Allocator>(
                 cost += ARITH_COST_PER_ARG as u64;
                 let blob = int_atom(&arg, "unknown op")?;
                 byte_count += blob.len() as u64;
-                check_cost(allocator, cost, max_cost)?;
+                check_cost(
+                    allocator,
+                    cost + byte_count as Cost / ARITH_COST_PER_LIMB_DIVIDER,
```

Comment:
it is integer, yes. I'm not sure why you ask about `(cost + byte_count)`, that's not the value being divided. But since we haven't added up all the bytes yet, this is an under estimate of the total cost. But as an under estimate, it's still useful as an early abort